composer.json

```diff
@@ -36,10 +32,10 @@
         }
     ],
     "require": {
-        "thunder/thunder-distribution": "~6.0@STABLE",
+        "thunder/thunder-distribution": "~6.3@STABLE",
         "drupal/core-composer-scaffold": "^9.0",
         "composer/installers": "^1.2",
-        "drush/drush": "^10.3",
+        "drush/drush": "^11.0",
```

Comment:
Thats not allow, thats require, allow would be
`        "drush/drush": "^10.3||^11.0",`
Do we even need drush as a requirement?

Reply:
IMHO, I think it's best practice to work with drush, so we should ship it.

Reply:
I dont know core/recommended-project does not. You might not need it in at least some of your containers?

Reply:
We are referencing to drush in many places all around our documentation. Would be strange to not have it by default

Reply:
Might be require-dev at max.

Reply:
Drush is used on production